nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -144,7 +149,7 @@ blocks:
 
   update:
     ops:
-      # aka findOneAndUpdate
+      # aka updateOne
```

Comment:
subtle change, but for apples to apples comparison with Docs API we should use variant that doesn't return the document.